### ISSUE TITLE
Loading log files using http/https protocol

### ISSUE THREAD

**Issue by littlesnitch** [zarunbal/LogExpert#73]
# Question
Could you please help me? I want to trigger log files which are hosted at an http-Link.
Do you know if anybody already was working on http protocol extension?

Important questions
- LogExpert 1.9.0
- Windows 10 1803
- Network
- Columnizer
- new problem

**Comment by zarunbal** [comment]
I'm currently not aware of any implementation.
Is this file static or continuously changing?

**Comment by littlesnitch** [comment]
Yes, this file continuously changing, because of that I wanted to use LogExpert for that because of the filters etc.

**Comment by jbaehr** [comment]
I think the main problem here is the same that currently prohibits direct reading from zip files (#92): The requirement for a seekable stream.

**Comment by igitur** [comment]
I also have a need for this to read streaming logs on Azure Functions. Currently I'm using `curl` with basic authentication. Getting this into LogExpert would greatly help so that I can apply my custom color rules and pause the logs when necessary. 

More info about Azure's streaming logs at https://github.com/projectkudu/kudu/wiki/Diagnostic-Log-Stream

**Comment by RandallFlagg** [comment]
@Hirogen can this be closed or maybe change/add enhance label?
